### PR TITLE
[MI-2959] Review fixes on Gitlab PR #306(Add create issue modal)

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
1. Fixed theme issue in post action menu.
2. Fixed the max length of issue title.
3. Added check to display the projects of connected gitlab group only.
4. Added check to display the issues of connected gitlab group only.

### CODE REVIEW

File: server/gitlab/api.go [hunk 1]
```diff
@@ -643,20 +643,41 @@ func (g *gitlab) GetYourProjects(ctx context.Context, user *UserInfo) ([]*intern
...
+		result, resp, err := client.Projects.ListProjects(
+			&internGitlab.ListProjectsOptions{
+				Owned: &owned,
+			},
+			internGitlab.WithContext(ctx),
+		)
+		if respErr := checkResponse(resp); respErr != nil {
+			return nil, respErr
+		}
+		if err != nil {
```
Comment (ayusht2810) [inline server/gitlab/api.go:657]:
```suggestion
		}
		
		if err != nil {
```

File: server/gitlab/api.go [hunk 2]
```diff
@@ -643,20 +643,41 @@ func (g *gitlab) GetYourProjects(ctx context.Context, user *UserInfo) ([]*intern
...
-		return nil, err
+	var projects []*internGitlab.Project
+	if g.gitlabGroup == "" {
+		result, resp, err := client.Projects.ListProjects(
+			&internGitlab.ListProjectsOptions{
+				Owned: &owned,
+			},
+			internGitlab.WithContext(ctx),
+		)
+		if respErr := checkResponse(resp); respErr != nil {
```
Comment (ayusht2810) [inline server/gitlab/api.go:654]:
```suggestion
		)
		
		if respErr := checkResponse(resp); respErr != nil {
```

File: server/gitlab/api.go [hunk 3]
```diff
@@ -643,20 +643,41 @@ func (g *gitlab) GetYourProjects(ctx context.Context, user *UserInfo) ([]*intern
...
+			return nil, respErr
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		projects = append(projects, result...)
```
Comment (ayusht2810) [inline server/gitlab/api.go:677]: this code is repeated. can we common this out somehow?

File: server/gitlab/api.go [hunk 4]
```diff
@@ -778,19 +799,39 @@ func (g *gitlab) SearchIssues(ctx context.Context, user *UserInfo, search string
...
+			internGitlab.WithContext(ctx),
+		)
+		if respErr := checkResponse(resp); respErr != nil {
+			return nil, respErr
+		}
+		if err != nil {
+			return nil, errors.Wrap(err, "can't search issues in GitLab api")
+		}
+
+		issues = append(issues, result...)
```
Comment (ayusht2810) [inline server/gitlab/api.go:831]: same

File: server/gitlab/api.go (hunk 3 above)
Comment (raghavaggarwal2308) [reply server/gitlab/api.go:677]: Discussed. This code is same in the whole plugin

**Review by ayusht2810 — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**